docs/jwst/assign_wcs/reference_files.rst

```diff
@@ -29,53 +29,132 @@ reftype                                     description
+
+IFUSLICER
+:::::::::
+For NIRSPEC, IFUSLICER reference files are currently selected based only on the value of EXP_TYPE in the input science data set.
+
+
+MSA
+:::
+For NIRSPEC, MSA reference files are currently selected based only on the value of EXP_TYPE in the input science data set.
+
+OTE
+:::
+For NIRSPEC, OTE reference files are currently selected based only on the value of EXP_TYPE in the input science data set.
+
+SPECWCS
+:::::::
+For MIRI, SPECWCS reference files are currently selected based on the values of DETECTOR, CHANNEL, BAND, SUBARRAY, and EXP_TYPE in the input science data set.
+
+For NIRCAM, SPECWCS reference files are currently selected based on teh values of EXPT_TYPE, MODULE, and PUPIL in the input science data set.
+
+For NIRISS GRISM and TSGRIM modes, SPECWCS reference files are currently selected based on the values of SUBARRAY and EXP_TYPE in the input science data set.
+
+FOR NIRISS WFSS mode, SPECWCS reference files are currently selected based on the values of EXP_TYPE, FILTER, and PUPIL in the input science data set.
+
+
+
+
+REGIONS
+:::::::
+For MIRI, REGIONS reference files are currently selected based on the values of DETECTOR, CHANNEL, BAND, EXP_TYPE in the input science data set.
+
+
+V2V3
+::::
+For MIRI, V2V3 reference files are currently selected based on the values of DETECTOR, CHANNEL, BAND, EXP_TYPE in the input science data set.
+
+
+WAVELENGTHRANGE
+:::::::::::::::
+For NIRSPEC, WAVELENGTHRANGE reference files are currently selected based only on the value of EXP_TYPE in the input science data set.
+
+For MIRI, WAVELENGTHRANGE reference files are currently selected based only on the value of EXP_TYPE in the input science data set.
+
+For NIRCAM, WAVELENGTHRANGE reference files are currently selected based only on the value of EXP_TYPE in the input science data set.
+
+For NIRISS, WAVELENGTHRANGE reference files are currently selected based only on the value of EXP_TYPE in the input science data set.
 
-:NIRSPEC: EXP_TYPE, GRATING
 
 
 Reference File Formats
-::::::::::::::::::::::
+----------------------
+
+
+CAMERA
+::::::
+
+The camera reference file contains an astropy compound model made up of a polynomial models, rotation, and translations. The forward direction is from the FPA to the GWA.
+
+:model: Transform through the CAMERA.
+
+COLLIMATOR
+::::::::::
+
+This collimator reference file contains an astropy compound model made up of a polynomial models, rotation, and translations. The forward direction is from the GWA to the MSA.
+
+:model: Transform through the COLLIMATOR.
+
+DISPERSER
+:::::::::
+
 
-The disperser file contains reference data about the NIRSPEC dispersers (gratings or the prism). The reference data is described in the NIRSPEC Interface Control Document.
+The disperser reference file contains reference data about the NIRSPEC dispersers (gratings or the prism). The reference data is described in the NIRSPEC Interface Control Document.
```

Comment:
Reference to the NIRSpec Interface Control Document?

Reply:
If someone has the reference or link to that, I'll add it, otherwise, I'll open as a separate issue.

Reply:
AFAIK it's not a public document. I only have a copy.
Perhaps remove this sentence and open a ticket to make sure our reference files are self documented, i.e. make sure the descriptions provided in the schemas are those in the ICD.

Reply:
Okay created #2227